pkg/services/ngalert/api/authorization.go

```diff
@@ -126,29 +126,48 @@ func (api *API) authorize(method, path string) web.Handler {
 	// convert/prometheus API paths
 	case http.MethodGet + "/api/convert/prometheus/config/v1/rules/{NamespaceTitle}/{Group}",
 		http.MethodGet + "/api/convert/prometheus/config/v1/rules/{NamespaceTitle}":
-		eval = ac.EvalAll(
-			ac.EvalPermission(ac.ActionAlertingRuleRead),
-			ac.EvalPermission(dashboards.ActionFoldersRead),
+		eval = ac.EvalAny(
```

Comment:
I replaced these with permissions for provisioning without detailed scopes for now. It still needs work to add the required scope resolver to find the UID of the resource by name.